DNN Platform/Modules/DnnExportImport/Components/Services/PagesExportService.cs

```diff
@@ -897,7 +897,7 @@ private int ImportTabModulesAndRelatedItems(TabInfo localTab, ExportTab otherTab
                                 local.DisplayTitle = other.DisplayTitle;
                                 local.DisplayPrint = other.DisplayPrint;
                                 local.DisplaySyndicate = other.DisplaySyndicate;
-                                local.IsDeleted = other.IsDeleted;
+                                //local.IsDeleted = other.IsDeleted;
```

Comment:
If a code element is no longer needed it should be removed, not commented out as we have history tracking available via Git.

Reply:
agree. my mistake to overlook.